PySDM/physics/constants_defaults.py

```diff
@@ -428,3 +428,5 @@ def compute_derived_values(c: dict):
 bulk_phase_partitioning_T_cold = 235 * si.K
 bulk_phase_partitioning_T_warm = 273 * si.K
 bulk_phase_partitioning_exponent = np.nan
+
+BOLIN_C1 = np.nan * si.dimensionless
```

Comment:
It would be worth to add a docstring here clarifying where one could read about the constants. These docstrings are then displayed here: https://open-atmos.github.io/PySDM/PySDM/physics/constants_defaults.html

We could also perhaps think of a more meaningful name for the constant, to indicate that it is isotope and timescale related?

NB: it seems that pdoc expects the docstrings to be just below a constant definition, while in this file we do it just above, hence the docstrings are in fact misleading!

Reply:
Maybe something like that?
`BOLIN_HTO_TIMESCALE_COEFF_C1`

"""Bolin 1958 coeffitient c1 used in calculations of the evaporation timescale for falling drop containing tritium (presented in Table 1)."""

Reply:
Yes, something along this line would greatly improve code readibility here. Since we do not include any value (thus asking the user to provide one), mentioning tritium might be actually misleading since the model will work for other isotopes; it would be worth, though, to mention that it is timescale of equilibration with ambient air void of a given isotopologue; also whenever mentioning a paper, best to do `[Bolin 1958](https://)` so that we will later be able to pick it for reference list